app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -420,4 +420,24 @@
   <dimen name="revision_card_fragment_layout_text_margin_top">32dp</dimen>
   <dimen name="revision_card_fragment_layout_button_margin_top">24dp</dimen>
 
+  <!-- HintsSummary -->
+  <dimen name="hint_summary_hint_summary_container_layout_marginTop">0dp</dimen>
+  <dimen name="hint_summary_hint_constraint_layout_marginStart">176dp</dimen>
+  <dimen name="hint_summary_hint_constraint_layout_marginEnd">176dp</dimen>
+  <dimen name="hint_summary_hint_constraint_layout_marginBottom">0dp</dimen>
+  <dimen name="hint_summary_hint_constraint_layout_paddingBottom">4dp</dimen>
+  <dimen name="hint_summary_hint_title_layout_marginTop">44dp</dimen>
+  <dimen name="hint_summary_hint_title_layout_marginBottom">0dp</dimen>
+  <dimen name="hint_summary_hint_title_layout_paddingBottom">28dp</dimen>
+  <dimen name="hint_summary_reveal_hint_button_layout_marginTop">0dp</dimen>
+  <dimen name="hint_summary_reveal_hint_button_layout_marginBottom">28dp</dimen>
+  <dimen name="hint_summary_expand_list_icon_layout_marginStart">0dp</dimen>
+  <dimen name="hint_summary_expand_list_icon_layout_marginEnd">0dp</dimen>
+  <dimen name="hint_summary_expand_list_icon_layout_marginTop">32dp</dimen>
+  <dimen name="hint_summary_expand_list_icon_layout_marginBottom">16dp</dimen>
+  <dimen name="hint_summary_expand_list_icon_layout_constraintHorizontal_bias">0.5</dimen>
+  <dimen name="hint_summary_expand_list_icon_layout_constraintVertical_bias">0.5</dimen>
+  <dimen name="hint_summary_hint_list_container_layout_marginStart">176dp</dimen>
+  <dimen name="hint_summary_hint_list_container_layout_marginEnd">176dp</dimen>
+  <dimen name="hint_summary_hint_list_container_layout_marginTop">0dp</dimen>
```

Comment:
@ArchitJain1201 newline above `</resources>` like it was before, also do it in all the dimens file too.